README.md

```diff
@@ -132,7 +132,55 @@ Task lock is implemented to prevent task cache collision.
     redis_port=6379
     ```
 
-### Inherit task parameters with decorator
+### 2. Using efficient task cache collision lock
```

Comment:
How about move these detailed documentations into `docs` directory?

Reply:
@Hi-king I've moved this descriptions to docs!